src/widgets/index/index.ts

```diff
@@ -419,29 +406,6 @@ const index = (props: IndexProps): Index => {
           });
         }
       });
-
-      // Hack for backward compatibily with `searchFunction` + `routing`
-      // https://github.com/algolia/instantsearch.js/blob/509513c0feafaad522f6f18d87a441559f4aa050/src/lib/RoutingManager.ts#L113-L130
-      if (localParent === null && isFirstRender) {
```

Comment:
i don't think this is expected to be removed now?

Reply:
It's replaced by https://github.com/algolia/instantsearch.js/compare/next...feat/fs-routingmanager?expand=1#diff-1fc36c4ea042d2ecb5a498416e5e0357R366-R369

I discussed it with @samouss and it should work the same way.

Reply:
Ahh that makes sense. How do I best test it?

Reply:
You can see this related story: https://deploy-preview-4099--instantsearchjs.netlify.com/stories/?path=/story/instantsearch--with-searchfunction-to-prevent-search
And this unit test: https://github.com/algolia/instantsearch.js/blob/d49ca2bc5caf5a7d85a5a331907766ba25f82743/src/lib/__tests__/RoutingManager-test.ts#L516-L565